cpp/src/robotick/framework/registry/TypeDescriptor.cpp

```
@@ -68,4 +67,172 @@ namespace robotick
 		return nullptr;
 	}
 
-} // namespace robotick
\ No newline at end of file
+	inline bool case_insensitive_equals(const char* a, const char* b)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)